api/src/chat/validation-rules/is-message.ts

```diff
@@ -12,103 +12,40 @@ import {
       return false;
-    } else {
-      return true;
     }
+    return true;
   } else if (Array.isArray(msg)) {
-    // Simple text message
-    const textSchema = Joi.array().items(Joi.string().max(1000).required());
-    const textCheck = textSchema.validate(msg);
-    return !textCheck.error;
-  } else if (typeof msg === 'object') {
-    if ('plugin' in msg) {
-      return true;
-    } else {
-      const buttonsSchema = Joi.array().items(
-        Joi.object().keys({
-          type: Joi.string().valid('postback', 'web_url').required(),
-          title: Joi.string().max(20),
-          payload: Joi.alternatives().conditional('type', {
-            is: 'postback',
-            then: Joi.string().max(1000).required(),
-            otherwise: Joi.forbidden(),
-          }),
-          url: Joi.alternatives().conditional('type', {
-            is: 'web_url',
-            then: Joi.string().uri(),
-            otherwise: Joi.forbidden(),
-          }),
-          messenger_extensions: Joi.alternatives().conditional('type', {
-            is: 'web_url',
-            then: Joi.boolean(),
-            otherwise: Joi.forbidden(),
-          }),
-          webview_height_ratio: Joi.alternatives().conditional('type', {
-            is: 'web_url',
-            then: Joi.string().valid('compact', 'tall', 'full'),
-            otherwise: Joi.forbidden(),
-          }),
-        }),
-      );
-      // Attachment message
-      const objectSchema = Joi.object().keys({
-        text: Joi.string().max(1000),
-        attachment: Joi.object().keys({
-          type: Joi.string()
-            .valid('image', 'audio', 'video', 'file', 'unknown')
-            .required(),
-          payload: Joi.object().keys({
-            url: Joi.string().uri(),
-            id: Joi.string().allow(null),
-          }),
-        }),
-        elements: Joi.boolean(),
-        cards: Joi.object().keys({
-          default_action: buttonsSchema.max(1),
-          buttons: buttonsSchema.max(3),
-        }),
-        buttons: buttonsSchema.max(3),
-        quickReplies: Joi.array()
-          .items(
-            Joi.object().keys({
-              content_type: Joi.string()
-                .valid('text', 'location', 'user_phone_number', 'user_email')
-                .required(),
-              title: Joi.alternatives().conditional('content_type', {
-                is: 'text',
-                then: Joi.string().max(20).required(),
-              }),
-              payload: Joi.alternatives().conditional('content_type', {
-                is: 'text',
-                then: Joi.string().max(1000).required(),
-              }),
-            }),
-          )
-          .max(11),
-      });
-      const objectCheck = objectSchema.validate(msg);
-      if (objectCheck.error) {
-        // eslint-disable-next-line
-        console.log('Message validation failed! ', objectCheck);
-      }
-      return !objectCheck.error;
+    const result = textSchema.safeParse(msg);
```

Comment:
Hi @abdou6666, can we face maybe an issue when the text introduced from the UI / created by a plugin  has a length > 1000